src/ui/StartUp.java

```diff
@@ -104,7 +96,7 @@ public static void main(String[] args) {
 			frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
 			frame.setTitle("Cognitive Bias Modifier");
 			frame.setResizable(false);
-			frame.add(new StartUp(4, 750, 30, new ImageManager()), BorderLayout.CENTER);
+			frame.add(new StartUp(4, 750, 30, new ImageManager(new int[12])), BorderLayout.CENTER);
```

Comment:
Problema ramane: daca cineva schimba cifra 4 sau 12 (si nu le schimba pe amandoua deodata) e de rau. Trebuie sa ma gandesc la o rezolvare :wink:

Reply:
Aici poate fi implementata o constrangere. O cifra sa se modifice in functie de cealalta, si astfel functioneaza corect programul.